templates/ts/custom-copilot-assistant-assistants-api/README.md.tpl

```diff
@@ -55,6 +68,31 @@ Before running or debugging your bot, please follow these steps to setup your ow
    SECRET_OPENAI_API_KEY=<your-openai-api-key>
    OPENAI_ASSISTANT_ID=<your-openai-assistant-id>
    ```
+{{/useOpenAI}}
+{{#useAzureOpenAI}}
+1. After `npm install` completed, run command `npm run assistant:create -- <your-azure-openai-api-key> <your-azure-openai-deployment-name> <your-azure-openai-endpoint>`
+   ```
+   > npm run assistant:create -- xxxxxx xxxxxx xxxxxx
+   ```
+1. The above command will output something like "*Created a new assistant with an ID of: **asst_xxx...***"
+1. Fill in both Azure OpenAI API Key and the created Assistant ID into `env/.env.*.user`
+   ```
+   SECRET_AZURE_OPENAI_API_KEY=<your-azure-openai-api-key>
+   AZURE_OPENAI_ENDPOINT=<your-azure-openai-endpoint>
+   AZURE_OPENAI_MODEL_DEPLOYMENT_NAME=<your-azure-openai-model-delopyment-name>
+   AZURE_OPENAI_ASSISTANT_ID=<your-azure-openai-assistant-id>
+   ```
+
+**If you already have an Assistant created**
+
+1. Fill in both Azure OpenAI API Key and the created Assistant ID into `env/.env.*.user`
```

Comment:
Fill in Azure OpenAI API Key, ednpoint, deployment name, and the created Assistant ID into env/.env.*.user